benchmarks/benchmark.js

```diff
@@ -10,7 +10,7 @@ const { WritableStream } = require('stream/web')
 
 const { Pool, Client, fetch, Agent, setGlobalDispatcher } = require('..')
 
-const iterations = (parseInt(process.env.SAMPLES, 10) || 100) + 1
+const iterations = (parseInt(process.env.SAMPLES, 10) || 10) + 1
```

Comment:
Why does it need to change? I have run it locally and it's working fine. When `10` it uses only 1 sample.

Reply:
Nothing, I committed my current branch.